ui/components/app/signature-request-original/signature-request-original.container.js

```diff
@@ -145,8 +140,7 @@ function mmiMapDispatchToProps(dispatch) {
         }),
       ),
     setWaitForConfirmDeepLinkDialog: (wait) =>
-      dispatch(mmiActions.setWaitForConfirmDeepLinkDialog(wait)),
```

Comment:
Same, are you sure we don't need the dispatch?

Reply:
Answered above 👍🏼